cli/flox-core/src/activations.rs

```diff
@@ -241,7 +239,7 @@ impl Activation {
     pub fn startup_process_running(&self) -> bool {
         self.attached_pids
             .first()
-            .map(|attached_pid| attached_pid.is_running())
+            .map(|attached_pid| pid_is_running(attached_pid.pid))
```

Comment:
(non-blocking, commentary) This is the only functional change which aligns us on using the implementation that we have more faith in 👍🏻